docs/developers/talawa-api/mutations.md

```diff
@@ -16,14 +18,16 @@ There are mainly 3 types of User types in talawa -
 3. `USER`
 
 ### Mutations Permitted by Roles.
+
 The limits of each role's use of mutations are stated below:
+
 #### `SUPERADMIN` -
 
-These are the Mutations exclusive to the `SUPERADMIN` type. Please note that a few of these mutations also have a few extra criteria with who can access them .For eg. except `createOrganization` mutation all other require that SUPERADMIN should also be the creator of the orgnanization.
+These are the Mutations exclusive to the `SUPERADMIN` type. Please note that a few of these mutations also have a few extra criteria with who can access them. For example, except `createOrganization` mutation all other require that SUPERADMIN should also be the creator of the orgnanization.
```

Comment:
Please change this paragraph to the following:
"Below are the Mutations exclusive to the `SUPERADMIN` type. Please note, a few of these mutations have a few extra criteria regarding user access. For example, except for the `createOrganization` mutation, all other mutations require the SUPERADMIN to also be the creator of the orgnanization."

Reply:
Done